SwSpotify/spotify.py

```diff
@@ -112,13 +112,35 @@ def get_info_mac():
     return a[3], a[1]
 
 
-def current():
-    if sys.platform.startswith("win"):
-        return get_info_windows()
-    elif sys.platform.startswith("darwin"):
-        return get_info_mac()
+def get_info_web():
+    from SwSpotify.web_data import WebData
+    import SwSpotify.web_server
+    import os
+    subprocess.Popen([sys.executable, os.path.join(os.path.dirname(__file__) + "/server_shutdown.py")])
+    SwSpotify.web_server.run()
+    if WebData.track == "":
+        raise SpotifyClosed
+    elif WebData.playState == "Play":
+        raise SpotifyPaused
     else:
-        return get_info_linux()
+        return WebData.track, WebData.artist
+
+
+def current():
+    try:
+        if sys.platform.startswith("win"):
+            return get_info_windows()
+        elif sys.platform.startswith("darwin"):
+            return get_info_mac()
+        else:
+            return get_info_linux()
+    except SpotifyPaused:
+        try:
+            return get_info_web()
+        except SpotifyClosed:
+            raise SpotifyPaused
+    except SpotifyClosed:
+        return get_info_web()
```

Comment:
well but what if Spotify Web is closed as well at this point? Maybe something like this would be more interesting https://github.com/SwagLyrics/SwSpotify/pull/12/files#diff-90dd5312bd6dd3addf7ce093dcad2283R160

Reply:
This seems better. Commit is on the way